JSDemos/Demos/Toolbar/Multiline/jQuery/index.js

```diff
@@ -0,0 +1,390 @@
+const toolbarDataDefault = [
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'undo',
+      onClick() {
+        DevExpress.ui.notify('Undo button has been clicked!');
+      },
+    },
+  },
+  {
+    location: 'before',
+    widget: 'dxButton',
+    options: {
+      icon: 'redo',
+      onClick() {
+        DevExpress.ui.notify('Redo button has been clicked!');
+      },
+    },
+  },
+  {
+    cssClass: 'toolbar-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $('<div>')
+        .addClass('toolbar-separator')
+        .appendTo(element);
+    },
+    menuItemTemplate(itemData, itemIndex, element) {
+      $('<div>')
+        .addClass('toolbar-menu-separator')
+        .appendTo(element);
+    },
+  },
+  {
+    location: 'before',
+    locateInMenu: 'auto',
+    widget: 'dxDropDownButton',
+    options: {
+      displayExpr: 'text',
+      keyExpr: 'size',
+      useSelectMode: true,
+      items: fontSizes,
+      selectedItemKey: 14,
+      itemTemplate(itemData) {
+        return $('<div>')
+          .text(itemData.text)
+          .css('font-size', `${itemData.size}px`);
+      },
+    },
+  },
+  {
+    location: 'before',
+    locateInMenu: 'auto',
+    widget: 'dxDropDownButton',
+    options: {
+      icon: 'indent',
+      displayExpr: 'text',
+      keyExpr: 'lineHeight',
+      useSelectMode: true,
+      items: lineHeights,
+      selectedItemKey: 1.35,
+      onSelectionChanged(e) {
+        $('#text').css('line-height', e.item.lineHeight);
+      },
+    },
+  },
+  {
+    cssClass: 'toolbar-separator-container',
+    locateInMenu: 'auto',
+    location: 'before',
+    template(itemData, itemIndex, element) {
+      $('<div>')
+        .addClass('toolbar-separator')
+        .appendTo(element);
+    },
+    menuItemTemplate(itemData, itemIndex, element) {
+      $('<div>')
+        .addClass('toolbar-menu-separator')
+        .appendTo(element);
+    },
+  },
+  {
+    locateInMenu: 'auto',
+    location: 'before',
+    widget: 'dxSelectBox',
+    options: {
+      placeholder: 'Font',
+      displayExpr: 'text',
+      dataSource: [
+        { text: 'Arial' },
+        { text: 'Courier New' },
+        { text: 'Georgia' },
+        { text: 'Impact' },
+        { text: 'Lucida Console' },
+        { text: 'Tahoma' },
+        { text: 'Times New Roman' },
```

Comment:
Why didn't u move this data to `data.js`? Let's fix it for all such places

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/05c2a4c6f2f88d68152ae92b286c4eba0e1284b7